docs/spec/spec.md

```diff
@@ -84,6 +75,7 @@ medical-center
 
 Behaviour of the `init` command,
 - User should invoke the command within a bal project
+- User can use the optional environment variables to indicate the preferred module name and data store, otherwise default values will be used.
```

Comment:
```suggestion
- User can use the optional arguments to indicate the preferred module name and data store, otherwise default values will be used.
```

Reply:
updated in [c6f8aa2](https://github.com/ballerina-platform/persist-tools/pull/148/commits/c6f8aa200f6cd6b2665e3f091a75d12c1ef59a0f)